package.json

```diff
@@ -17,7 +17,11 @@
     "test:integration": "pnpm -r --filter=./integration-tests/* run test",
     "test:types": "pnpm -r --filter=./packages/* run test:types",
     "test": "pnpm -r --filter=./packages/* run test",
-    "typesync": "pnpm exec typesync && pnpm -r exec typesync && pnpm install"
+    "typesync": "pnpm exec typesync && pnpm -r exec typesync && pnpm install",
+    "format": "prettier --write 'src/**/*.js'",
```

Comment:
This repo is typescript so the glob doesn't look right to me :sweat_smile:   

Did you test this command locally? Did it do anything?